src/__tests__/test-a11y-validator.test.tsx

```diff
@@ -25,9 +25,10 @@ describe('a11y validator', () => {
         <input id="my-field" type="text" />
       </label>
     );
-    // TODO: add <div> element is not permitted as content under <label> error when element-permitted-content is enabled
     await expect(expect(container).toValidateA11y()).rejects.toThrow(
-      'Expected HTML to be valid but had the following errors:\n1. <label> is associated with multiple controls [multiple-labeled-controls]'
```

Comment:
String argument checks if error message includes the substring. Need to pass error obj to ensure if checks for all violations